README.md

```diff
@@ -6,6 +6,10 @@ This is the [JHipster](http://www.jhipster.tech/) registry service, based on [Sp
 
 Full documentation is available on the [JHipster documentation for microservices](http://www.jhipster.tech/microservices-architecture).
 
+##Pre-requirements
+
+ 1. plz make sure that nodejs version is between 6 and 9 (I have test it in node8)
```

Comment:
Please change to `Make sure to that your NodeJS version is the latest LTS version supported by JHipster`

Reply:
I think the specified version is better.  We could not work based on the latest node10.

Reply:
You need an LTS version - otherwise Node is far too buggy for any serious use

Reply:
Please make the suggested correction. We only support and recommend latest LTS version. Node 10 is not LTS yet